OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -828,6 +841,29 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     thread_name.set_timestamp_ns(event->GetTimestamp());
     listener_->OnThreadName(std::move(thread_name));
 
+  } else if (is_user_instrumented_tracepoint) {
+    auto it = ids_to_tracepoint_info_.find(stream_id);
+
+    if (it != ids_to_tracepoint_info_.end()) {
```

Comment:
I would write this with an early return to avoid indenting everything:

```
if (it == ids_to_tracepoint_info_.end()) {
  return;
}

auto event = ConsumeGenericTracepointPerfEvent(ring_buffer, header);
...
...
```